File: app/src/main/java/com/dvor/my/mydvor/NotificationsFakeActivity.java
package com.dvor.my.mydvor;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class NotificationsFakeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_notifications_fake);

        Intent i;
        i = new Intent(this, MainActivity.class);
        i.putExtra("fragment", "NotificationFragment");
        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
        startActivity(i);
    }
}

File: app/src/main/java/com/dvor/my/mydvor/NotificationsFakeActivity.kt
package com.dvor.my.mydvor

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class NotificationsFakeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_notifications_fake)

        val i = Intent(this, MainActivity::class.java)
        i.putExtra("fragment", "NotificationFragment")
        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK)
        startActivity(i)
    }
}